Dockerfile

```diff
@@ -15,3 +15,8 @@ RUN /tmp/buildpack/php/bin/compile /app /tmp/build_cache /tmp/env
 
 # Set up xdebug
 RUN $HEROKU_PHP_BIN/pecl install channel://pecl.php.net/xdebug-2.4.0
+
+# Set up Redis
+RUN wget http://download.redis.io/releases/redis-4.0.11.tar.gz
```

Comment:
+1, it be better to use an off the shelf Redis container and then compose.

The Redis version probably doesn't matter too much since Workbench is using very old/basic level functionality. Of course, version matching would be nice, but shouldn't be a big priority.